modules/apps/headless/headless-portal-instances/headless-portal-instances-test/src/testIntegration/java/com/liferay/headless/portal/instances/resource/v1_0/test/PortalInstanceResourceTest.java

```diff
@@ -91,7 +93,13 @@ public void testPatchPortalInstance() throws Exception {
 	@Test
 	public void testPostPortalInstance() throws Exception {
 		_testPostPortalInstanceWithoutAdmin();
-		_testPostPortalInstanceWithAdmin();
+
+		try (SafeCloseable safeCloseable =
+				PropsValuesTestUtil.swapWithSafeCloseable(
```

Comment:
@javalosjr96 Please avoid using this method when other ways are possible. This `PropsValues.COMPANY_SECURITY_STRANGERS_VERIFY` is used as the fallback value in `CompanySecurityBag`, which firstly queries `PrefsPropsUtil` and then `PropsUtil` with `PropsKeys.COMPANY_SECURITY_STRANGERS_VERIFY`.

Please try to swap the value in the `PropsUtil` level to avoid reflection usage.

Also, for coverage purpose, I would like to see `_testPostPortalInstanceWithAdmin()` executed with and without this property swapped.

Reply:
@javalosjr96 See relevant discussion about the reflection usage: https://liferay.slack.com/archives/C01FP01V5L0/p1729091655543689

We should avoid using it as much as possible.

Reply:
got it.